packages/drawer/src/Drawer.tsx

```diff
@@ -0,0 +1,149 @@
+import React, {
+  useRef, useContext, useState, useCallback,
+} from 'react';
+import { withTheme, ThemeContext, ThemeProvider } from 'styled-components';
+import { TransitionGroup, Transition } from 'react-transition-group';
+
+import { IThemeNamespace } from '@xcritical/theme';
+import Portal from '@xcritical/portal';
+import Blanket from '@xcritical/blanket';
+
+import { IDrawerProps, DrawerTheme } from './interfaces';
+import {
+  Wrapper,
+  Content,
+  Fade,
+  Slide,
+  SeparatorWrapper,
+  Separator,
+  AntiSelect,
+  IconWrapper,
+} from './styled';
+import { ArrowLeft, ArrowRight } from './Icons';
+
+
+export const PureDrawer: React.FC<IDrawerProps> = React.memo<IDrawerProps>(({
+  children,
+  appearance = 'default',
+  baseAppearance = 'default',
+  isOpen = false,
+  onOutsideClick,
+  theme,
+  minWidth = 30,
+  maxWidth = 1000,
+  isRTL = false,
+  isMovable = false,
+  withCloseButton = false,
+}: IDrawerProps) => {
+  const themeContext = useContext<IThemeNamespace<DrawerTheme>>(ThemeContext);
+
+  const clickX = useRef(0);
+  const drawerRef = useRef<HTMLElement>(null);
+
+  const [width, setWidth] = useState(maxWidth);
+  const [antiSelectLayer, setAntiSelectLayer] = useState(false);
+
+
+  const handleMouseMove = useCallback(
+    (e) => {
+      if (!isMovable) return;
+
+      setAntiSelectLayer(true);
+      document.body.addEventListener('mouseup', () => {
```

Comment:
You should add this event listener inside handleMouseDown and remove it inside handleMouseUp.
You could see example here: xc-front-kit\packages\grid\src\HeaderCell.tsx

Reply:
We have not 'handleMouseUp' method

Reply:
No, it is not work, I have checked it